src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseVB.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+
+using FluentAssertions;
+
+using Xunit.Abstractions;
+using Xunit;
+using System;
+using System.IO;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToUseVB : SdkTest
+    {
+        public GivenThatWeWantToUseVB(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        private enum VBRuntime
+        {
+            Unknown,
+            Default,
+            Embedded,
+            Referenced
+        }
+
+        [Theory]
+        [InlineData("net45")]
+        [InlineData("netstandard2.0")]
+        [InlineData("netcoreapp2.1")]
+        [InlineData("netcoreapp3.0")]
+        public void It_builds_a_simple_project(string targetFramework)
+        {
+            var (expectedVBRuntime, expectedOutputFiles) = GetExpectedOutputs(targetFramework);
+
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("HelloWorldVB", identifier: targetFramework)
+                .WithSource()
+                .WithProjectChanges(project =>
+                {
+                    var ns = project.Root.Name.Namespace;
+                    var propertyGroup = project.Root.Elements(ns + "PropertyGroup").First();
+                    propertyGroup.Element(ns + "TargetFramework").SetValue(targetFramework);
+                })
+                .Restore(Log);
+
+            var buildCommand = new GetValuesCommand(
+                Log,
+                testAsset.TestRoot,
+                targetFramework,
+                "VBRuntime")
+            {
+                DependsOnTargets = "Build"
+            };
+
+            buildCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var outputDirectory = buildCommand.GetOutputDirectory(targetFramework);
+            var actualVBRuntime = GetVBRuntime(buildCommand.GetValues().FirstOrDefault());
+            File.Delete(outputDirectory.File("VBRuntimeValues.txt").FullName);
+
+            outputDirectory.Should().OnlyHaveFiles(expectedOutputFiles);
+            actualVBRuntime.Should().Be(expectedVBRuntime);
+        }
+
+        private static(VBRuntime, string[]) GetExpectedOutputs(string targetFramework)
```

Comment:
self-nit: missing space

```suggestion
        private static (VBRuntime, string[]) GetExpectedOutputs(string targetFramework)
```